### ISSUE TITLE
Refactor Alaveteli banner HTML and CSS

### ISSUE THREAD

**Issue by wrightmartin** [mysociety/alaveteli#3644]
It needs a rethink and refactor

- There's loads of different styles and JS that all do the same thing
- The UX is inconsistent between menus (language switcher, navigation menu and the new pro menu)
- The CSS is over-complicated and hard to follow

**Comment by garethrees** [comment]
I'm going to close this and say that we should refactor when we come to work on the banner.

It might mean that "small" changes will need extra time to account for the rework, but I don't think this is such a problem that we need to do it as an individual step.
